src/fdb5/api/FDB.cc

```diff
@@ -325,6 +331,19 @@ void FDB::registerCallback(ArchiveCallback callback) {
     internal_->registerCallback(callback);
 }
 
+void FDB::registerCallback(FlushCallback callback) {
+    flushCallback_ = callback;
+}
+
+void FDB::initPlugins(const Config& config){
+    bool enableGribjump = eckit::Resource<bool>("fdbEnableGribjump;$FDB_ENABLE_GRIBJUMP", false);
```

Comment:
I do not want to see the words gribJump in the FDB (or really even the word GRIB)